Packages/com.unity.inputsystem/InputSystem/NativeInputRuntime.cs

```diff
@@ -264,31 +264,31 @@ public Action onProjectChange
                     EditorApplication.projectChanged += OnProjectChanged;
                 m_OnProjectChanged = value;
             }
-        }
-
+        }
+
         #endif // UNITY_EDITOR
-
-        #if UNITY_ANALYTICS || UNITY_EDITOR
-
+
         public void RegisterAnalyticsEvent(string name, int maxPerHour, int maxPropertiesPerEvent)
-        {
+        {
+            #if UNITY_ANALYTICS
```

Comment:
Wondering if the above #if should just be turned into `#if UNITY_ANALYTICS` (dropping the editor case) instead.

IIRC my intention here was to send analytics from the editor even if the project has it disabled thinking that whether the user has analytics disabled for his/her own game shouldn't matter in the editor. But I'm probably just mixing up stuff here and that's not how it works.

If `UNITY_ANALYTICS` not being defined should cull all analytics code, we should probably just drop the `UNITY_EDITOR` part from the `#if UNITY_ANALYTICS || UNITY_EDITOR` defines everywhere.

Reply:
Yeah, AFAIK, this will not compile, even in the editor, when targeting a platform without analytics, hence the change.

Reply:
Just to clarify is the change OK? I took it from what @sambickley recommended as he found the problem when testing on PS4.